book/tutorials_examples/2_HBV_PCRGlobWB_ERA5/example_model_run_pcrglobwb.ipynb

```diff
@@ -18,46 +18,86 @@
   },
```

Comment:
<p>let's talk about this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/projects/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>